src/olympia/addons/tests/test_tasks.py

```diff
@@ -256,7 +256,7 @@ def test_flag_high_hotness_according_to_review_tier():
         name='D tier (below minimum usage for hotness)',
         lower_adu_threshold=0,
         upper_adu_threshold=100,
```

Comment:
I would think we want to cover cases where growth is negative? just so it is included in the set of possible scenarios?

Reply:
https://github.com/mozilla/addons-server/pull/22875/commits/b0a647158306cc65637d80ad5e73548bd91e428e